model-api/bpmn-model/src/main/java/org/camunda/bpm/model/bpmn/builder/AbstractCallActivityBuilder.java

```diff
@@ -156,6 +156,18 @@ public B camundaCaseTenantId(String tenantId) {
     return myself;
   }
 
+  /**
+   * Sets the businessKey
+   * @param businessKey the business key to set
+   * @return the builder object
+   */
+  public B camundaBusinessKey(String businessKey) {
```

Comment:
```suggestion
  public B camundaInBusinessKey(String businessKey) {
```
Can we rename this API method to make the relation to `CamundaIn` more obvious?

Reply:
Renamed